### DISCUSSION TITLE
VStack divider is being applied for null child

### DISCUSSION THREAD

**Discussion by carlosberti** [chakra-ui/chakra-ui#7409] [Q&A]
Not sure if that is supposed to be the correct behavior, but the divider is being applied 'after' a null child. You can see the images below. The first divider, I think it should not be there

<img width="578" alt="image" src="https://user-images.githubusercontent.com/22118601/222190886-13aaeeaf-5676-4330-9570-debbf589c753.png">

![image](https://user-images.githubusercontent.com/22118601/222191290-27582034-1587-4ca9-aaf9-53ecf4e4d37f.png)

**Comment by intuity-hans** [comment]
Just ran into the same problem. I conditionally render each child in the list and Dividers are added in between.
@carlosberti : did you find a solution to this?

**Reply by carlosberti** [reply]
Not actually. The problem is the way that they are dealing with they call "validChild". If you go to the "stack" file at theirs repository, you will see that they do not filter the items to remove null ones. The only solution that I found was to filter the items before add them to the stack. Another possible solution it would be to open a pull request and see if that's really a problem or if it should work like this

**Comment by sergeyyarkov** [comment]
same here = )
